### ISSUE TITLE
Review the Code of Conduct

### ISSUE THREAD

**Issue by terichadbourne** [ProtoSchool/hangzhou#2]
We've provided a draft Code of Conduct for you that includes the contact information you provided for reporting purposes. Please ensure that these contact details are correct.

Please close this issue after **all** chapter organizers have reviewed the CoC thoroughly and confirmed they are comfortable enforcing it.

(This will need to be done before we can list your chapter on our website.)

**Optional:** Our tutorials are currently all in English, but If you decide that you'll be conducting your chapter events and repo discussions in a language other than English, you are welcome to update your Code of Conduct to the appropriate language. What's used right now is a custom Code of Conduct that mixes the [Contributor Covenant](https://www.contributor-covenant.org/version/1/4/code-of-conduct) ([see available translations](https://www.contributor-covenant.org/translations)) with the [Conference Code of Conduct](http://confcodeofconduct.com/) ([see available translations](https://github.com/confcodeofconduct/confcodeofconduct.com)) to ensure it covers interactions both online and at events.

**Comment by rjl493456442** [comment]
The CoC looks good to me. Could you guys take another look? @yyh1102 @grokcoder @terryding77 @QibingLee @wcy1231

**Comment by grokcoder** [comment]
LGTM

**Comment by rjl493456442** [comment]
@yyh1102 @terryding77 @QibingLee @wcy1231 Please give me a thumb if you think it's fine for you.

**Comment by rjl493456442** [comment]
I'll close this issue now. If you guys have any questions about the CoC, please open another issue. Thanks.
